fractions/javaee/remoting/README.adoc

```diff
@@ -0,0 +1,4 @@
+# Remoting
+
+Primarily an internal fraction providing remote invocation
```

Comment:
> Primarily and internal fraction

That's pretty vague, IMO. What does that mean for the user? Applies to all fractions with a similar description.